src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/converter/LongToInstantConverter.java

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2023 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.northbound.converter;
+
+import java.time.Instant;
+
+/**
+ * This class converts Linux epoch seconds or milliseconds to an Instant object.
+ */
+public final class LongToInstantConverter {
+    private LongToInstantConverter() {
+    }
+
+    /**
+     * Converts Linux epoch seconds or milliseconds to an Instant object.
+     * @param timeAsLong non-null value that represents a Linux epoch time in seconds or milliseconds
+     * @return an Instant object
+     */
+    public static Instant convert(Long timeAsLong) {
+        if (timeAsLong == null) {
+            throw new IllegalArgumentException();
+        }
+
+        if (Math.log10(Math.abs(timeAsLong)) < 10) {
```

Comment:
Sure. Epoch time could contain different number of digits, depending on units of measure. In our case, we could have either seconds or milliseconds. That is: 1692171571 or 1692171571123. For a couple of centuries from now epoch time in seconds will have log base 10 less than 10, epoch time in milliseconds will give log base 10 greater than 10 (to simplify log gets us the number of digits). Log defined only for positive numbers, so we need to take abs just in case; epoch time could be negative, but for our purposes it's actually not important. 
So this is not a universal converter, but when epoch time is around now and contains seconds or milliseconds, it'll be fine.

Reply:
I think it should be in comment near the code

Reply:
I updated the javadoc